From: Alexis Murzeau <amubtdx@gmail.com>
Date: Sat, 20 May 2023 18:36:45 +0200
Subject: docs: remove furo version specification in docs-requirements.txt

Packaged version of furo in Debian might not be the exact same version
as specified in docs-requirements.txt.

Forwarded: not-needed
Author: Alexis Murzeau <amubtdx@gmail.com>
---
 docs-requirements.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs-requirements.txt b/docs-requirements.txt
index 78e878d..cf154a3 100644
--- a/docs-requirements.txt
+++ b/docs-requirements.txt
@@ -1,5 +1,5 @@
 sphinx >=5.0.0, <7
-furo ==2023.03.27
+furo
 myst-parser >=1.0.0, <2
 sphinx-design >=0.4.1, <1
 
